### PR TITLE
fix pep8 W503

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: base_partner_merge/base_partner_merge.py [hunk 1]
```diff
@@ -371,16 +371,16 @@ def _merge(self, cr, uid, partner_ids, dst_partner=None, context=None):
...
-                                  ).search(cr, openerp.SUPERUSER_ID,
-                                           [('partner_id',
-                                             'in',
-                                             [partner.id for partner
-                                              in src_partners])],
-                                           context=context)):
+        if (openerp.SUPERUSER_ID != uid and
+                self._model_is_installed(cr, uid, 'account.move.line',
+                                         context=context) and
+                self.pool.get('account.move.line'
```
Comment (yvaucher) [inline base_partner_merge/base_partner_merge.py:377]: self.pool[] as we are already changing this line?

Comment (gurneyalex) [inline base_partner_merge/base_partner_merge.py:377]: not too keen in this. The goal here is to get a green 8.0 branch, not to fix bad code everywhere.

I'd implement that in a separate PR.

**Comment by pedrobaeza** [comment]
Haven't you seen my PR? https://github.com/OCA/partner-contact/pull/89

It fixes both things.

**Comment by pedrobaeza** [comment]
I close this one in favor of #89

**Pull request closed without merging**